Lib/site.py

```diff
@@ -335,11 +335,17 @@ def getsitepackages(prefixes=None):
         seen.add(prefix)
 
         if os.sep == '/':
+            if sys.python_libdir != "lib":
+                sitepackages.append(os.path.join(prefix, sys.python_libdir,
+                                            "python%d.%d" % sys.version_info[:2],
+                                            "site-packages"))
             sitepackages.append(os.path.join(prefix, "lib",
                                         "python%d.%d" % sys.version_info[:2],
                                         "site-packages"))
```

Comment:
```suggestion
            libdirs = ["lib"]
            if sys.python_libdir != "lib":
                libdirs.insert(0, sys.python_libdir)
            for libdir in libdirs:
                dir = os.path.join(prefix, libdir,
                                        "python%d.%d" % sys.version_info[:2],
                                        "site-packages")
            sitepackages.append(dir)
```

Same for the `else` block to avoid the copy-and-paste of the path creation.

Reply:
Right. I merged duplicated code based on your proposal, thanks.